components/Text.js

```diff
@@ -10,6 +10,7 @@ export const P = styled.p.attrs(props => ({
 }))`
   ${color}
   ${display}
+  ${layout}
```

Comment:
Ok. I will try this approach and update the PR.

Reply:
@Betree I tried this approach. Unfortunately, it does not work, with both `Box`, `Container`. I think the problem with this approach is the surrounding container adds the height to all the cards, with and without description. This again leads to spacing issues where cards without description is shown without a min height. 

There is another solution, which is more design centric where a `Box` is added surrounding the `Name`, `ADMIN label`, `date` and `description` with a fixed height. But it involves some change to layout and some design effort. 

If you have any ideas, please let me know.

Reply:
@Betree  Digging more into this, this problem requires a layout change. The problem is primarily caused by `margin-bottom` for the `description`, while `description` itself does not have any fixed height. 

The original PR solution will also break down if the description overflows to multiple lines. I have attached a screenshot of this behaviour. 
![Screenshot 2022-06-02 at 11 18 03 AM](https://user-images.githubusercontent.com/4299398/171561764-40e1f6f5-d8bf-476a-8b4d-b65ebaa4d921.png)

I think a better approach is to have a fixed height for description. Otherwise when there are cards with description of varying sizes, this problem will recur.

Reply:
Following works, which adds a fixed height `72px` to description (which can be changed)
```
          <Box mb={5} overflow="hidden" height={72}>
            <P fontSize="11px" lineHeight="16px" mx={2} minHeight={16} fontWeight={400}>
              {get(member, 'description')}
            </P>
          </Box>
```

The only caveat is long description will be clipped. In current behaviour, description will not get clipped but will cause card height issues. 

![Screenshot 2022-06-02 at 11 46 42 AM](https://user-images.githubusercontent.com/4299398/171565426-7a8c99b3-bc89-46ec-b046-bdcd9d889bde.png)

If this looks acceptable, please let me know and I will update the PR.

Reply:
Thanks for digging and providing these clear explanations! The last solution you proposed looks good. On the collective page, we actually truncate this text to `30`, so maybe we can do that here?

```jsx
import { truncate } from 'lodash';

<P title={description} ...>{truncate(description, { length: 30 })}</P>
```

Reply:
> we actually truncate this text to 30, so maybe we can do that here?

I have updated the PR with this suggestion. I have also reduced the fixed height of description to `32` (since text is truncated to 30 characters.)

Attached screenshot with three possible cases shown together - no description, short description (< 30 chars), long description (> 30 chars)

![Screenshot 2022-06-03 at 1 27 15 PM](https://user-images.githubusercontent.com/4299398/171813683-b56c71c3-504b-469c-a84c-0a6bcfd3b5c8.png)

Reply:
@Betree Does the current change looks good for fixing this issue? If anything more is needed for this PR, please let me know.